var/spack/repos/builtin/packages/hipsycl/package.py

```diff
@@ -36,6 +37,11 @@ class Hipsycl(CMakePackage):
 
     depends_on("cmake@3.5:", type="build")
     depends_on("boost +filesystem")
+
+    # TODO: replace this with an explicit list of components of Boost,
+    # for instance depends_on('boost +filesystem')
+    # See https://github.com/spack/spack/pull/22303 for reference
+    depends_on(Boost.with_default_variants)
```

Comment:
ping @nazavode to confirm boost components used by hipsycl

Reply:
Looks like this to me: https://github.com/illuhad/hipSYCL/blob/develop/CMakeLists.txt#L35
- context
- fiber
and for tests:
- unit_test_framework

Reply:
Hi @ax3l, thanks for the ping. The current version we are providing via spack depends on `filesystem` and `preprocessor` (plus `test` and `mpl` when unit tests are enabled, something we are not exposing to users right now). `context` and `fiber` will be needed with the shiny brand new version of hipsycl.

Reply:
#22770 introduces correct deps on `boost` components for newer versions too, looks like `hipsycl` is ready for this PR.